scripts/changelog.sh

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env bash
+#
+# Copyright 2017 The rkt Authors
+# Copyright 2020 Kinvolk
+#
+# Create a changelog.
+#
+# The env variable RANGE specifies the range of commits to be searched for the changelog.
+# If unset the latest tag until origin/master will be set.
+#
+# The env variable GITHUB_AUTH can be set in the form user:token to specify a GitHub
+# personal access token. Otherwise one could run into GitHub rate limits.
+# Go to https://github.com/settings/tokens to generate a token.
+
+set -e
+
+jq --version >/dev/null 2>&1 || {
+    echo "could not find jq (JSON command line processor), is it installed?"
+    exit 255
+}
+
+if [ -z "${RANGE}" ]; then
+    LATEST_TAG=$(git describe --abbrev=0)
+    RANGE="${LATEST_TAG}..origin/master"
+fi
+
+if [ ! -z "${GITHUB_AUTH}" ]; then
+    GITHUB_AUTH="-u ${GITHUB_AUTH}"
+fi
+
+for pr in $(git log --pretty=%s --first-parent "${RANGE}" | egrep -o '#\w+' | tr -d '#'); do
+    body=$(curl -s "${GITHUB_AUTH}" https://api.github.com/repos/kinvolk/lokomotive/pulls/"${pr}" | \
```

Comment:
Should this be called with GITHUB_TOKEN as well to avoid hitting rate limits?

Reply:
Good point, will fix.

Reply:
The script already supports passing the token as part of the `GITHUB_AUTH` env variable, which should be set to `user:token`. I can make it more consistent with the rest of the release process and just ask for `GITHUB_TOKEN`.

Reply:
Yep, sounds good :+1: